compose/frontend.yml

```diff
@@ -83,13 +83,15 @@ services:
       PROXY_DEFINITION_IMPORT: http://ccd-definition-store-api:4451
       PROXY_DEFINITION_DATA: http://ccd-definition-store-api:4451/api/data
       PROXY_DEFINITION_DISPLAY: http://ccd-definition-store-api:4451/api/display
+      PROXY_CASE_DOCUMENT: http://ccd-case-document-am-api:4455
       APPINSIGHTS_INSTRUMENTATIONKEY: key
     ports:
       - 3453:3453
     depends_on:
       - ccd-user-profile-api
       - ccd-definition-store-api
       - ccd-data-store-api
+      - ccd-case-document-am-api
```

Comment:
Would this make the case-document a mandatory service rather than an optional one tha can be enabled separately, if so is that what we want?

Reply:
I took it to be a mandatory service, given the following comments in the JIRA ticket

> Configure the ccd-docker project to run the Case Document API when the backend is enabled.
> 
> Acceptance Criteria:
> 
> When the command - ./ccd enable backend frontend - is executed, the case document api project started up along with other existing projects.

Reply:
I agree with Matt, `ccd-case-document-am-api` should not be a dependency of ccd-api-gateway. 
Ideally every service could be started up by default. The problem is memory footprint, which is already very high. So I'd say we can make ccd-case-document-am-api optional. Also considering that by default we CCD Docker starts the CCD UI, and the CCD UI won't make use of it.
`ccd-case-document-am-api ` needs to be a dependency of `xui-manage-cases` instead, meaning that if the EUI is enabled, then the new service needs to be started as well. 
What do you think?

Reply:
makes sense

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/119/commits/013bf7fcf6ecdbed6b3910dfc483196caf41b9c4